server/src/main/resources/org/jboss/as/server/controller/descriptions/LocalDescriptions.properties

```diff
@@ -223,9 +223,22 @@ deployment.enabled=Boolean indicating whether the deployment content is currentl
 deployment.persistent=Boolean indicating whether the existence of the deployment should be recorded in the persistent server configuration. Only relevant to a standalone mode server. Default is 'true'. A deployment managed by a deployment scanner would have this set to 'false' to ensure the deployment is only deployed at server start if the scanner again detects the deployment.
 deployment.owner=Address of a resource that has indicated it controls the lifecycle of this deployment (e.g. a deployment scanner resource.) Will be undefined if no resource claimed ownership when the deployment was added.
 deployment.add=Adds previously uploaded content to the list of content available for use. Does not actually deploy the content unless the 'enabled' parameter is 'true'.
+deployment.add-content=Add contents to an existing deployment.
+deployment.add-content=Add contents to an existing deployment.
+deployment.add-content.overwrite=Iondicates if added content should overwrite existing content.
+deployment.add-content.content=List of pieces of content to be added to the deployment.
+deployment.browse-content=List the pieces of content of a deployment.
+deployment.browse-content.path=The relative path from a deployment to list the contents of.
+deployment.browse-content.archive=To return only the archive file relative paths.
+deployment.browse-content.depth=The depth to browse.
 deployment.deploy=Deploy the specified deployment content into the runtime, optionally replacing existing content.
 deployment.deploy.name=The name of the new content.
 deployment.deploy.to-replace=The name of the content that is to be replaced.
+deployment.empty=Indicates that the deployment to be added is empty - so without any content.
+deployment.read-content=Read the content of an existing deployment.
+deployment.read-content.path=The relative path of the content to be read from an existing deployment.
+deployment.remove-content=Remove contents from an existing deployment.
+deployment.remove-content.path=List of paths of content to be removed from the deployment.
```

Comment:
Should this param be 'paths' since it can be many? For add-content, the name of param 'content' doesn't imply only 1 but 'path' does.